player.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <string>
+
+class Player
+{
+    std::string name;
+    unsigned short hp;
+    unsigned short damage;
+
+public:
+    explicit Player(std::string name, unsigned short hp, unsigned short damage);
+    bool Attack(Player* otherPlayer);
+    void Print() const;
+    std::string GetName() const { return name; };
```

Comment:
Visszatérési értékben is inkább const referencia legyen. És az attack nem lett const, pedig lehetne.